config/src/comments.rs

```diff
@@ -210,11 +210,40 @@ fn comments() -> HashMap<String, String> {
 		.to_string(),
 	);
 
+	retval.insert(
+		"i2p_mode".to_string(),
+		"
+#Configuration for I2P support (\"Disabled\" by default)
+#I2P can be enabled by the following:
+#[server.p2p_config.i2p_mode]
+#mode = \"Enabled\"
+
+#[server.p2p_config.i2p_mode.i2p_config]
+#autostart = true/false  - start I2P automatically
+#exclusive = true/false  - exclusively connect through I2P, or also use TCP
+#addr = \"127.0.0.1:7656\"  - address of local I2P server
+"
+		.to_string(),
+	);
+
 	retval.insert(
 		"[server.p2p_config.capabilities]".to_string(),
 		"#If the seeding type is List, the list of peers to connect to can
-#be specified as follows:
-#seeds = [\"192.168.0.1:3414\",\"192.168.0.2:3414\"]
+#be specified as follows (same for other peer types):
+#For TCP addresses:
```

Comment:
These are IP addresses, TCP is how the packages are transmitted between the addresses in the network. If I2P and TCP are mutually exclusive protocols and that's what you want to call out here then maybe use "TCP/IP" instead of "TCP"?